src/utils/macros.rs

```diff
@@ -5,168 +5,184 @@ macro_rules! csi {
     ($( $l:expr ),*) => { concat!("\x1B[", $( $l ),*) };
 }
 
-/// Write a string to standard output whereafter the stdout will be flushed.
+/// Writes a string to the given writer.
 #[doc(hidden)]
 #[macro_export]
-macro_rules! write_cout {
+macro_rules! write_string {
     ($write:expr, $string:expr) => {{
```

Comment:
* `$write:expr` should be `$writer:expr`
* `$string:expr` should be `$ansi_code:expr`